openshift-knative-operator/cmd/operator/kodata/ingress/0.25/0-networkpolicy-mesh.yaml

```diff
@@ -16,15 +16,15 @@ spec:
 apiVersion: networking.k8s.io/v1
 kind: NetworkPolicy
 metadata:
-  name: istio-webhook
```

Comment:
@markusthoemmes I've no clue here so I'm guessing that's planned? I saw a patch removing the "net" for kourier stuff so just making sure

Reply:
Yep this is expected and shouldn't impact anything.